Tests/Common/ZipFileChecker.cs

```diff
@@ -0,0 +1,51 @@
+//-----------------------------------------------------------------------
+// <copyright file="ZipFileChecker.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
Existing helper class: moved and renamed from _JarChecker_ to _ZipFileChecker_.
I've changed all of refs in the class from "jar" to "zip".